### ISSUE TITLE
Look into focus issue with drawer + nav tabs

### ISSUE THREAD

**Issue by silvalaura** [cengage/react-magma#948]
In the magma docs, when the hamburger menu is clicked (small screens), the focus should immediately be on the first item on the list (Introduction). Instead, the user has to click "tab" once for the focus to get on the drawer and then tab through the list of options.

We should look into this, as it may help us find a solution for adopters who can replicate this issue.

**Comment by ccedrone** [comment]
**For Testing**

Two parts:

Documentation site - ensure after site loads that a focus goes to the next item below Introduction (Contribution Guidelines) on mobile.

Documentation site - go to `components` > `Drawer` > Navigation Example, click "Show Drawer", verify that first element is selected in the list.

**Comment by Uditsingh67** [comment]
The focus is not on Contribution Guidelines after the Site is loaded.
See Screen-cast : http://somup.com/c3Xu6fwXwR
Hence, Reopening the ticket.
Thanks!

**Comment by ccedrone** [comment]
Sorry, the initial focus should be on Introduction, then after tabbing, the focus goes to Contribution Guidelines. Please re-test.

**Comment by Uditsingh67** [comment]
The Issue is fixed.
See Screen-cast : http://somup.com/c3XUhjwlWp
Thanks!
